gradle/jacoco/build.gradle

```diff
@@ -14,9 +14,9 @@ jacocoTestReport {
     }
     dependsOn test
     reports {
-        xml.enabled true
-        html.enabled false
-        csv.enabled false
+        xml.required.set(true)
+        html.required.set(true)
+        csv.required.set(true)
```

Comment:
why setting these to true?

Reply:
My bad, seems I overlooked it, will change these to false.
Also pushing a new commit will slight changes.